### ISSUE TITLE
Parsing the value Orientation for Image metadata throws an error

### ISSUE THREAD

**Issue by vigitri** [gettyimages/gettyimages-api_dotnet#40]
SDK Request:
var imageMetadata = await client.Images().WithId("someId").ExecuteAsync();

Json Response:
```
{
    "images": [
        {
            "...":"..."
            "orientation": "PanoramicHorizontal",
        }
    ],
    "images_not_found": []
}
```

Trying to parse the EnumValue PanoramicHorizontal throws an error as the enum doesn't contain this value
```
'The JSON value could not be converted to GettyImages.Api.Models.OrientationImages. Path: $.images[0].orientation | LineNumber: 0 | BytePositionInLine: 6478.'
```

```
namespace GettyImages.Api.Models
{
  [Flags]
  public enum OrientationImages
  {
    None = 0,
    Horizontal = 1,
    Panoramic_Horizontal = 2,
    Panoramic_Vertical = 4,
    Square = 8,
    Vertical = 16, // 0x00000010
  }
}
```

**Comment by mapitman** [comment]
We'll look at fixing this soon. If you'd like to make the change and submit a pull request, we'd welcome that as well.

**Comment by ssterli2** [comment]
This bug has been fixed with the release of 8.0.1: https://www.nuget.org/packages/GettyImages.Api/8.0.1
